examples/cache.ts

```diff
@@ -0,0 +1,23 @@
+import { cache } from '../lib/cache/';
+
+class Service {
+  @cache(1000, { scope: 'class' })
+  public f(n: number): Promise<number> {
```

Comment:
Remember, [*Clarity is King*](https://github.com/labs42io/clean-code-typescript#avoid-mental-mapping).
Please provide a more meaningful names than `f` & `g`.